_test/test_converters/test_detpar_default_detector.m

```diff
@@ -0,0 +1,73 @@
+classdef test_detpar_default_detector < TestCase
+    
+    properties
+        detpar_single = struct( ...
+            'width', 54, ...
+            'height', 32, ...
+            'group', 11, ...
+            'phi', 9.8, ...
+            'x2', 7.6, ...
+            'azim', 5.4);
+        
+        detpar_multi = struct( ...
+            'width', 123.45, ...
+            'height', 987.65, ...
+            'group', [26 31 41 59 53], ...
+            'phi', [3.1 4.1 5.9 2.6 5.3], ...
+            'x2', [1 1 2 3 5], ...
+            'azim', [1 2 4 8 16]);
+    end
+    
+    methods
+        
+        %% Default Detector
+        function this = test_default_detector_is_He3Tube(this)
+            detector = get_default_detector_from_detpar(this.detpar_single);
+            
+            assertEqual(numel(detector), 1);
+            assertEqual(class(detector), 'IX_det_He3tube');
```

Comment:
I would use `isa` here, it's similar to using `isintance()` rather `type() ==` in Python
```suggestion
            assertTrue(isa(detector, 'IX_det_He3tube');
```

There's another instance of this in the test below and in `test_detpar_default_detector_array.m`